Code/client/ModCompat/BehaviorVar.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#ifdef MODDED_BEHAVIOR_COMPATIBILITY
+
+#include <Structs/AnimationGraphDescriptorManager.h>
+
+struct BehaviorVar
+{
+    struct Replacer
+    {
+        uint64_t origHash;
+        uint64_t newHash;
+        std::string signatureVar;
+        std::string creatureName;
+        std::vector<std::string> syncBooleanVar;
+        std::vector<std::string> syncFloatVar;
+        std::vector<std::string> syncIntegerVar;
+        bool operator==(const uint64_t& acRHS) const
+        {
+            return origHash == acRHS;
+        }
+    };
+
+    static BehaviorVar* Get();
+    const AnimationGraphDescriptor* Patch(BSAnimationGraphManager* apManager, Actor* apActor);
+    boolean failListed(const uint64_t acHash);
```

Comment:
bool?

Reply:
Pushed the clang-format commit. Reverted the "disable MODDED_ANIMATION_COMMIT".

So I think this is ready for re-review and merge.